### PR TITLE
Login design

### LINKED ISSUE #1: 1) Feature: New design for login/signup pages
We’ve received new specs from our designer. Please implement the design for the login and signup pages, and make it mobile responsive. The design spec and needed assets are available in [this zip folder](https://storage.googleapis.com/hatchways-app.appspot.com/assessments/data/project-overview/4f81e627-c085-4583-a9ef-bf69316a7a55/assets%202.zip). The spec may need a bit of adapting, so please use your best judgement.

Please include a couple of screenshots in the description when making this PR to showcase the styling you've done.

### PR DESCRIPTION
**Pull request by tianhaoyao**
Implemented login and register with styling as per mockup. Linked to issue #1

### CODE REVIEW

**Review by WhoaNellie — COMMENTED** [review]
Couple of notes!
- With UI Changes, it's helpful to add screenshots of what the UI looks like to your PR description so reviews can see the style without having to run the code!

- There's a good amount of repeated code/styles between the Login and Signup pages, you should consider what you can make into reusable components

- There's a [theme.js](https://github.com/tianhaoyao/chat-app/blob/43827468e4f6b646b38b1097c4c0cf56223db2b5/client/src/themes/theme.js) defined for this app, you should try to add reusable color/font/etc. values to the theme and using them in your makeStyles to make the styling more maintainable!

**Comment by tianhaoyao** [comment]
refactor
register
login

Screenshots attached are what the login and signup pages, and how I refactored to avoid duplicate code . I tried to utilize reusable colors/such a bit, not sure if it's enough.

File: client/src/SignupForm.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    height: '100vh',
+    padding: '0 30px',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  button: {
+    maxWidth: '200px',
+    maxHeight: '70px',
+    minWidth: '200px',
+    minHeight: '70px',
```
Comment (tmattmac) [inline client/src/SignupForm.js]: You can just set `width` and `height` instead here.

File: client/src/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                      label="Password"
+                      aria-label="password"
+                      type="password"
+                      name="password"
+                      InputProps={{endAdornment:
+                      <Button>
+                        <Typography variant="h6" color="primary" disableFocusRipple disableRipple>
+                          Forgot?
+                        </Typography>
+                      </Button>}
```
Comment (tmattmac) [inline client/src/LoginForm.js]: You don't have to include the "Forgot" button, but in the future, you should wrap this in an [`InputAdornment`](https://material-ui.com/components/text-fields/#input-adornments) component.

**Review by tmattmac — COMMENTED** [review]
Changes look good, just a few notes:

Make sure you're consistently using classes instead of inline styles.

The buttons are a bit large; the text should be about the size of the text in the input fields.

Also, I noticed that some of the indenting for the JSX isn't lining up. It's recommended to use a code formatter like Prettier if you find it troublesome to keep track of indentation.

I'm also noticing a lot of extra spaces between lines. Adding line breaks within functions/components is fine to distinguish connected blocks of code, but try to avoid using more than one line break where necessary, and avoid doing things like this:

image

**Comment by tianhaoyao** [comment]
login
register

I have shrunk the button text sizes to match the input text size, removed the forgot password button, moved inline styles to classes, and formatted everything with Prettier. Used width and height instead of max/min width/height.

**Review by bonnieli — APPROVED** [review]

**Pull request merged**